README.md

```diff
@@ -2,17 +2,57 @@ This repository accompanies [the Networking Test Pyramid blog](https://www.inten
 
 # Running the test suite
 
-1. Download and run Batfish. Details [here](https://pybatfish.readthedocs.io/en/latest/getting_started.html); short version:
+## Setup
+1. Run the Batfish service. Details [here](https://pybatfish.readthedocs.io/en/latest/getting_started.html); short version:
    - `docker pull batfish/allinone`
    - `docker run --name batfish -v batfish-data:/data -p 8888:8888 -p 9997:9997 -p 9996:9996 batfish/allinone`
-3. Clone this repo
-4. In a Python virtual environment: 
+3. Clone this repo:
+   - ` git clone git@github.com:intentionet/test-pyramid.git`
+4. Install dependencies. In the top-level folder where you cloned the repo, and ideally in a Python virtual environment, do: 
    - `pip install -r requirements.txt`
-   - `pytest test_suite`
 
-# Repository organization
+
+## Running on the example network
+
+This repo contains an example network that you can use to play with the test suite. 
+
+ - You can run the test suite on this network by running the following command in the top-level folder:
+
+   `pytest test_suite`
+
+   All tests should pass.
+
+ - If you wanted to run tests at only one level of the pyramid, you can do: 
+
+   `pytest -k config_content test_suite`
```

Comment:
> -k config_content
How can users find the valid values for the `-k` flag and what each of them means?

Reply:
done